pulp_container/app/modelresource.py

```diff
@@ -76,6 +76,51 @@ class Meta:
         model = ManifestListManifest
 
 
+class BinaryDataWidget(widgets.Widget):
+    """
+    A widget that handles conversion for binary data between exported and imported objects.
+    """
+
+    def clean(self, value, row=None, *args, **kwargs):
+        """
+        Return a newly initialized memoryview object from a hexadecimal value.
+        """
+        return memoryview(bytearray.fromhex(value))
+
+    def render(self, value, obj=None):
+        """
+        Return the hexadecimal representation suitable for the exportable format.
+        """
+        return value.hex()
+
+
+class ManifestSignatureResource(BaseContentResource):
+    """
+    A resource for import/export of manifest signatures.
+    """
+
+    signed_manifest = fields.Field(
+        column_name="signed_manifest",
+        attribute="signed_manifest",
+        widget=widgets.ForeignKeyWidget(Manifest, field="digest"),
+    )
+
+    data = fields.Field(column_name="data", attribute="data", widget=BinaryDataWidget())
+
+    def set_up_queryset(self):
+        """
+        Return signatures specific to a specified repo-version.
+        """
+        return ManifestSignature.objects.filter(pk__in=self.repo_version.content).order_by(
+            "content_ptr_id"
+        )
+
+    class Meta:
+        model = ManifestSignature
+        import_id_fields = model.natural_key_fields()
+        skip_diff = True
```

Comment:
What does it imply? I'm not sure I understand what diffs bring, why are they enabled by default.
This https://django-import-export.readthedocs.io/en/latest/api_resources.html?highlight=skip_diff#import_export.resources.ResourceOptions.skip_diff does not help me.

Reply:
This implies that the content imported to Pulp will not be compared with the original content before the import. If I understand it correctly, the following [test](https://github.com/django-import-export/django-import-export/blob/0d4340f02582e86d4515159becb045cad1d53ee0/import_export/resources.py#L705) will not be executed:
```
parsed row before insert == parsed row after insert
```

Signatures will not be valid if the content will be corrupted after the import.

Reply:
@ipanova , @lubosmj , I realized that the signature is stored in the binary field because it was originally implemented to store the signature in a binary form but then we started storing it in the base64 encoded way.
Does it make sense to switch to the TextField then? It makes more sense now I think, and it should make import/export more robust it seems.
Thoughts?

Reply:
Yes, if we are storing everything in base64, we can convert it to `TextField` and I can therefore get rid of `skip_diff`. Either way, I initially planned to convert `BinaryField` data to its hexadecimal representation before export and then convert it back to the binary format during the import.

Reply:
Cool, let's do it in this PR then, so the import/export works as normal with diffs. Thanks!